src/modules/form/components/RecordPickerDialog.tsx

```diff
@@ -71,6 +71,8 @@ const RecordPickerDialog = ({
   }, [item]);
 
   // Need to set height on the dialog in order for the scrolling to work
+  // Columns are really rows here because this is a vertical table.
+  // The height should be between 550 and 850, with the # of columns determining where it falls within that.
   const height = `${Math.min(Math.max(columns.length * 60 + 250, 550), 850)}px`;
```

Comment:
Ok this REALLY was overly complicated and brittle, that's my bad. I think I did it this way because the designs showed that the table scrolled, but the pagination at the bottom was sticky. 

I think the actual regression came with change behavior for the top toolbar, because that wasn't there before. If you add `noSort` to the table it starts to look sane again.

Aaanyway.. I think we should switch to something more sustainable that doesn't rely on pixels like this. Something like https://github.com/greenriver/hmis-frontend/commit/90f229a46856da8f68f8aae495905d2d23618007 would make it so we just scroll the whole dialog, not just the table. I think that will be more robust to future changes to the table.

Reply:
Ah gotcha. Thanks for digging up the history. I like the simplification. I guess the downside is that now the pagination buttons jump around a bit when you move from page to page, depending on the line lengths of the records. Maybe that is not a big enough deal to warrant any more complex (brittle) styling.

Reply:
Ok, https://github.com/greenriver/hmis-frontend/pull/777/commits/974d1332be40a1162f6f0e196003224fb3712855 shows a suggestion for another way - possibly this is still too brittle, but it does have the advantage of keeping the dialog's height consistent as you page through the paginated records!

To be perfectly honest, that `190px` on line 114 was a matter of guess and check 😬

Reply:
Hmm, I still think we should remove the hard-coded pixels because it will be brittle. I'm also seeing 2 scrollbars here, that feels jumpy because when you scroll down it switches from the inner scroll to the outer scroll. I think to keep this simpler for the future, we should just get rid of the inner scroll.

<img width="273" alt="Screenshot 2024-05-21 at 6 51 35 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/c77ff08e-8ac8-4a8e-ac23-9d2cde59a73a">
<img width="273" alt="Screenshot 2024-05-21 at 6 52 07 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/6aa116c4-ce30-4ceb-a5bc-d71e6a0385fb">

Reply:
Ok good plan, you're right this was too brittle. I reverted to the version where the height may be inconsistent depending on the height of the rows within the table - maybe a minorly annoying ux but certainly better than 2 scroll bars!